src/pages/besquare/components/sections/_what-is_besquare.js

```diff
@@ -37,11 +37,11 @@ const WhatIsBeSquare = () => {
     return (
         <Section>
             <ContentContainer>
-                <ContentWrapper>
-                    <Title as="h2" text_align={'left'} max_width={['282px']} margin={['0', '0 auto']}>
+                <ContentWrapper margin={'120px'}>
```

Comment:
```suggestion
                <ContentWrapper>
```

Reply:
This wont affect ContentWrapper since there is no props for margin added for ContentWrapper component.

Reply:
i've done it few minutes ago)